ecs/netsil-collectors.json

```diff
@@ -0,0 +1,111 @@
+{
+   "executionRoleArn":null,
+   "containerDefinitions":[
+      {
+         "dnsSearchDomains":null,
+         "logConfiguration":null,
+         "entryPoint":null,
+         "portMappings":[
+
+         ],
+         "command":null,
+         "linuxParameters":{
+            "capabilities":{
+               "add":[
+                  "NET_RAW",
+                  "NET_ADMIN"
+               ],
+               "drop":null
+            },
+            "sharedMemorySize":null,
+            "tmpfs":null,
+            "devices":null,
+            "initProcessEnabled":null
+         },
+         "cpu":100,
+         "environment":[
+            {
+               "name":"SD_BACKEND",
+               "value":"docker"
+            },
+            {
+               "name":"DEPLOY_ENV",
+               "value":"docker"
+            },
+            {
+               "name":"NETSIL_ORGANIZATION_ID",
+               "value":${organizationId}
```

Comment:
https://trello.com/c/W8BZq8T9/2702-template-ecs-specs-in-the-docs for making this embedded